code/modules/clothing/under/marine_uniform.dm

```diff
@@ -8,6 +8,7 @@
 	icon_state = "marine_jumpsuit"
 	icon = 'icons/obj/items/clothing/uniforms/uniforms_by_map/jungle.dmi'
 	worn_state = "marine_jumpsuit"
+	w_class = SIZE_LARGE
```

Comment:
This will probably cause more issues then it solves, you can just make a check so if the clothes have an uniform attached you cant put them in the bag, or apply an uniform if theyre in a bag

Reply:
you can just do a !ishuman(I.loc) return check on the attackby on uniform